src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedTryDepthCheck.java

```diff
@@ -48,7 +48,27 @@
  *   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * try { // ...
+ *     try { // ...
+ *         try { // violation
+ *         } catch (Exception e) { // violation
+ *         }
+ *     } catch (Exception e) { // on Exception
+ *     }
+ * } catch (Exception e) { // on Exception
```

Comment:
Please remove redundant comments like `// on Exception` or `// ...`

Reply:
Should I add one `// ...` and `// on exception` on each code examples.